pkg/nanolib/signal/README.md

```
@@ -393,6 +393,32 @@ The `subscribe` method accepts an optional second argument to customize its beha
 - **`.untilNext()`**: Returns a `Promise` that resolves with the next value/payload.
 - **`.destroy()`**: (On all but `StateSignal`) Cleans up the signal.
 
+---
+
+## 🌊 Part of Alwatr Flux
+
+`@alwatr/signal` is the **State Layer** of the [Alwatr Flux](https://github.com/Alwatr/alwatr/tree/next/pkg/flux) architecture — a complete Unidirectional Data Flow system for building scalable Progressive Web Applications.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)